content/en/docs/refguide/modeling/pages/page/_index.md

```diff
@@ -94,10 +94,38 @@ In **Structure mode**, there are several ways to add an element on a page:
 
     1. Select an element you would like to add and confirm your choice by clicking **Select**.
 
-In **Design mode**, you can add elements though the Toolbox. Do the following:
+In **Design mode**, you can add elements through the Toolbox and UI Recommender. Do the following:
 
-1. Open the **Toolbox** and select **Widgets** or **Building blocks** tab there. 
-2. Select an element you would like to add and drag this element onto your page.
+1. Through the **Toolbox**:
+   1. Open the **Toolbox** and select **Widgets** or **Building blocks** tab there. 
+   1. Select an element you would like to add and drag this element onto your page.
+
+2. Through the **UI Recommender**:
+    1. Opening the Dialog
+
+        1. When hovering over an element on the page, a blue plus/dot will appear:
```

Comment:
It might make sense to call it an indicator and mention that the indicator is a blue plus or a dot

Reply:
Updated